core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -1,3 +1,7 @@
+registry.ci.openshift.org/origin/4.11:agent-installer-api quay.io/openshift/origin-agent-installer-api:4.11 quay.io/openshift/origin-agent-installer-api:4.11.0 quay.io/openshift/origin-agent-installer-api:latest
+registry.ci.openshift.org/origin/4.11:agent-installer-agent quay.io/openshift/origin-agent-installer-agent:4.11 quay.io/openshift/origin-agent-installer-agent:4.11.0 quay.io/openshift/origin-agent-installer-agent:latest
+registry.ci.openshift.org/origin/4.11:agent-installer-csr-approver quay.io/openshift/origin-agent-installer-csr-approver:4.11 quay.io/openshift/origin-agent-installer-csr-approver:4.11.0 quay.io/openshift/origin-agent-installer-csr-approver:latest
+registry.ci.openshift.org/origin/4.11:agent-installer-installer quay.io/openshift/origin-agent-installer-installer:4.11 quay.io/openshift/origin-agent-installer-installer:4.11.0 quay.io/openshift/origin-agent-installer-installer:latest
```

Comment:
agent-installer-client?
(again, just to try to avoid stuttering in the name)

Reply:
it is "installer" because of it being the process that drives the coreos-installer driver and performs other node operations. Having client does not convey the same degree of participation in the actual installation of the node.

Reply:
agent-installer-node-installer?  (if its logic is really constrained to running on/configuring one node, and not the larger cluster?)

Reply:
agent-installer-orchestrator ?

Reply:
🤔 Yeah I think this will be fine to describe what it does. I'll take it.